.github/workflows/release.yml

```diff
@@ -0,0 +1,109 @@
+name: kapp-controller release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  kapp-controller-release:
+    name: kapp-controller release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+        with:
+          path: src/github.com/${{ github.repository }}
+
+      - name: Install Carvel Tools
+        uses: vmware-tanzu/carvel-setup-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          only: ytt, kbld
+          ytt: v0.35.1
+          kbld: v0.31.0
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run release script
+        run: |
+          set -e -x
+
+          ./hack/build-release.sh
+
+      - name: Create Release with goreleaser
+        uses: goreleaser/goreleaser-action@5e15885530fb01d81d1f24e8a6f54ebbd0fed7eb
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
How do we feel on using token with these permissions? Is there a more specific one for pushing to ghcr.io.